tfhe/src/core_crypto/gpu/entities/lwe_ciphertext_list.rs

```diff
@@ -220,4 +220,19 @@ impl<T: UnsignedInteger> CudaLweCiphertextList<T> {
     pub(crate) fn ciphertext_modulus(&self) -> CiphertextModulus<T> {
         self.0.ciphertext_modulus
     }
+
+    /// # Safety
+    ///
+    /// - `stream` __must__ be synchronized to guarantee computation has finished, and inputs must
+    ///   not be dropped until stream is synchronised
+    pub unsafe fn set_to_zero_async(&mut self, streams: &CudaStreams) {
+        self.0.d_vec.memset_async(T::ZERO, streams, 0);
```

Comment:
here you could call with 0u64 directly ?

Reply:
With a comment explaining why it's ok

Side question : you don't have an as_ref/as_mut equivalent for cuda entities allowing access to the inner memory if required ?

Reply:
We have it for RadixCiphertexts but we didn't add it at the core crypto level.

Reply:
Ah I didn't think of using 0u64 directly, will do :pray:

Reply:
Ah no memset_async takes a type T that has to implement Into<u64> actually... I could maybe change the memset_async entry point itself?

Reply:
u64 should be Into<u64>

Reply:
as for the entry point change I don't know if it's worth it

Reply:
Looks much better